runtime/peaq/src/lib.rs

```diff
@@ -583,7 +583,7 @@ impl pallet_evm::GasWeightMapping for PeaqGasWeightMapping {
 }
 
 parameter_types! {
-	pub const ChainId: u64 = 42424242;
```

Comment:
If the 42424242 won't conflict with others, let us keep it the same. Because Leo chose this specific number based on some special meaning.

https://www.scientificamerican.com/article/for-math-fans-a-hitchhikers-guide-to-the-number-42/